conda-store-server/conda_store_server/app.py

```diff
@@ -352,14 +352,14 @@ def _docker_base_image(build: orm.Build):
 
     @property
     def session_factory(self):
-        if hasattr(self, "_session_factory"):
-            return self._session_factory
-
-        self._session_factory = orm.new_session_factory(
```

Comment:
Because this calls our orm wrapper, this also create a new engine each time. The engine needs to be created once.